### PR TITLE
Create custom 404 page

### PR DESCRIPTION
**Pull request by NullableInt**

### CODE REVIEW

**Comment by NullableInt** [comment]
Rubocop fails and I do not know why. 
`cannot load such file -- rubocop-performance`

File: 404.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+permalink: /404.html
+---
+
+<!-- markdownlint-disable MD026 -->
+# Uh oh:heavy_exclamation_mark: Seems like you have taken a wrong turn somewhere :construction:
```
Comment (payex-dev) [inline 404.md:6]: Codacy found an issue: [[maximum-heading-length] Use headings shorter than `60`](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=6230013)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=6230013&bid=20302553)

**Comment by asbjornu** [comment]
> Rubocop fails and I do not know why.
> `cannot load such file -- rubocop-performance`

I think I've fixed that in #150. But I can't quite recollect how.

**Comment by NullableInt** [comment]
> 
> 
> > Rubocop fails and I do not know why.
> > `cannot load such file -- rubocop-performance`
> 
> I think I've fixed that in #150. But I can't quite recollect how.

Perhaps it's the renaming from .rubycop.yml to .robocop.yml. But I have both files on my branch.

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=6230013&bid=20302553)

File: 404.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
```
Comment (payex-dev) [inline 404.md:6]: Codacy found an issue: [[maximum-heading-length] Use headings shorter than `60`](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=6230013)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=6230013&bid=20302553)

File: 404.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
```
Comment (payex-dev) [inline 404.md:6]: Codacy found an issue: [[maximum-heading-length] Use headings shorter than `60`](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=6230013)

**Comment by asbjornu** [comment]
> Perhaps it's the renaming from .rubycop.yml to .robocop.yml. But I have both files on my branch.

You should not have `.rubycop.yml` as it's been renamed to `.robocop.yml`. Delete.

**Review by asbjornu — APPROVED** [review]

**Comment by NullableInt** [comment]
> 
> 
> > Perhaps it's the renaming from .rubycop.yml to .robocop.yml. But I have both files on my branch.
> 
> You should not have `.rubycop.yml` as it's been renamed to `.robocop.yml`. Delete.

File has been deleted and will be merged promptly.

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=6230013&bid=20302553)

File: 404.md (hunk 3 above)
Comment (payex-dev) [inline 404.md:6]: Codacy found an issue: [[maximum-heading-length] Use headings shorter than `60`](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=6230013)

**Pull request merged**